examples/send_message_ble.rs

```diff
@@ -0,0 +1,235 @@
+//! Example: send_message_ble.rs
+//!
+//! This example demonstrates how to use the Meshtastic Rust API to send a message over
+//! a Bluetooth Low Energy (BLE) connection and checking for its ACK.
+//!
+//! Usage:
+//!    cargo run --example send_message_ble --features="bluetooth-le tokio" -- <DESTINATION_NODE | BROADCAST> <MESSAGE> [BLE_DEVICE_NAME]
+//!
+//! If the BLE_DEVICE_NAME is not provided, the example will scan and prompt to specify a device if multiple
+//! are found.
+//!
+//! if the DESTINATION_NODE is not found the example will dump all destination nodes found.
+
+use std::collections::{BTreeMap, VecDeque};
+use std::convert::Infallible;
+use std::io::Write;
+use std::time::Duration;
+
+use meshtastic::api::StreamApi;
+use meshtastic::packet::{PacketDestination, PacketRouter};
+use meshtastic::protobufs::from_radio::PayloadVariant;
+use meshtastic::protobufs::{mesh_packet, Data, MyNodeInfo, User};
+use meshtastic::protobufs::{FromRadio, MeshPacket, PortNum};
+use meshtastic::types::{MeshChannel, NodeId};
+use meshtastic::utils::generate_rand_id;
+use meshtastic::utils::stream::{build_ble_stream, BleId};
+
+use meshtastic::Message;
+
+struct Router {
+    sent: VecDeque<MeshPacket>,
```

Comment:
Is `sent` used for anything in the example, or do you just want to show that the router is informed when messages are sent by send_text() ?